Resources/doc/schema.md

```diff
@@ -49,6 +49,10 @@ The plugin can override the main file definitions of the Bower packages. To over
 definitions specify the packages and their main file array as name/value pairs. For an example
 see the [usage informations](index.md#override-the-main-files-for-bower).
 
+##### config.fxp-asset.git-driver.skip-update (root-only)
```

Comment:
Restore the config key `config.fxp-asset.git-skip-update`, and add an example of package config and explain this feature and its benefits.

```json
{
    "config": {
        "fxp-asset": {
            "git-skip-update": "3 days"
        }
    }
}
```